packages/node-utils/src/edge-to-node/stream.ts

```diff
@@ -0,0 +1,76 @@
+import { Readable } from 'node:stream'
+
+interface FromWebOptions {
+  objectMode?: boolean
+  highWaterMark?: number
+  encoding?: BufferEncoding
+  signal?: AbortSignal
+}
+
+/**
+ * Code adapted from Node's stream.Readable.fromWeb(), because it has to run on Node@14
+ * @see https://github.com/nodejs/node/blob/bd462ad81bc30e547e52e699ee3b6fa3d7c882c9/lib/internal/webstreams/adapters.js#L458
+ */
+export function toToReadable(
+  webStream: ReadableStream,
+  options: FromWebOptions = {}
+) {
+  const reader = webStream.getReader()
+  let closed = false
+  const { highWaterMark, encoding, objectMode = false, signal } = options
+
+  const readable = new Readable({
+    objectMode,
+    highWaterMark,
+    encoding,
+    // @ts-ignore signal exist only since Node@17
+    signal,
+    read() {
+      reader.read().then(
+        (chunk: any) => {
+          if (chunk.done) {
+            readable.push(null)
+          } else {
+            readable.push(chunk.value)
+          }
+        },
+        (error: any) => readable.destroy(error)
```

Comment:
Shouldn't we set the `callback` argument for `destroy` method?

Reply:
I'm confused: node's [Readable.destroy()](https://nodejs.org/api/stream.html#readabledestroyerror) does not have a callback.

In fairness, I've almost-blindly copied this from [node source](https://github.com/nodejs/node/blob/bd462ad81bc30e547e52e699ee3b6fa3d7c882c9/lib/internal/webstreams/adapters.js#L458).
This is the implementation of `Readable.fromWeb()`, which was only introduced in Node 16+, and that we can't use directly because this package must support Node 14+.

Reply:
I thought this calls `destroy` defined at line 41. LGTM thanks! 👍